.travis.yml

```diff
@@ -50,7 +50,7 @@ before_install:
   - echo "always_populate_raw_post_data = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```

Comment:
These travis updates are incomplete.  There should be changes at the top of the file for sudo=true.  
https://github.com/acquia/blt/blob/8.x/scripts/travis/.travis.yml

I would delete the file.  Regenerate it and then reconcile.